compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/fuse_logical_objectfifo_into_workgroup.mlir

```diff
@@ -0,0 +1,34 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-amdaie-fuse-logicalobjectfifo-into-workgroup)" %s | FileCheck %s
+
+// CHECK-LABEL: @fuse_logical_objectfifo_into_workgroup
+// CHECK:     %[[ALLOC0:.*]] = memref.alloc() : memref<32x1024xi32>
+// CHECK:     %[[ALLOC1:.*]] = memref.alloc() : memref<32x64xi32, 1>
+// CHECK:     scf.forall
```

Comment:
maybe a CHECK-NOT on amdaie.logicalobjectfifo.from_memref before the scf.forall to show they were infact erased?

Reply:
Yeah, good check to add. Thanks and added it.